README.md

```diff
@@ -7,29 +7,16 @@ First check wether the file for the provider already exists, if it does skip to
 Add [provider name].md to the _providers folder in the following format:
 ~~~
 ---
+website: [website of the provider]
 name: [provider name]
```

Comment:
nitpicking: i would leave name as the first item

Reply:
order does not matter in yaml AFAIK.

Reply:
True, in YAML the order does not matter, but in the README we're writing for humans, not the YAML-parser. So it makes some sense to maintain an order of fields.

Reply:
meh don't only look at the readme, it's the smallest part of the pr, also we can make it pretty later.

Reply:
I'm waiting with reviewing the code until it's settled and cleanly committed. Until then I'm looking at text.